core/management/commands/generate_data.py

```diff
@@ -0,0 +1,96 @@
+import random
+
+from django.core.management import BaseCommand
+from faker import Faker
+
+from posts.models import Post
+from spaces.models import Space
+from users.models import User
+from users.tests.factories import UserFactory
+
+
+class Command(BaseCommand):
+    help = "Creates a random local data for testing."
+
+    def handle(self, *args, **kwargs):
+        for _ in range(10):
+            UserFactory()
+
+        space_names = [
+            "Cat Memes",
+            "Tech Geeks",
+            "Pizza Lovers",
+            "Coffee Addicts",
+        ]
+
+        users = User.objects.all()
+
+        for name in space_names:
+            space, _ = Space.objects.get_or_create(name=name, owner=random.choice(users))
+            space.members.add(*users)
+            space.moderators.add(random.choice(users))
+            space.save()
+
+        spaces = Space.objects.all()
+
+        fake = Faker()
+
+        for _ in range(4):
+            user = User(
+                email=fake.email(),
+                username=fake.user_name(),
+                first_name=fake.first_name(),
+                last_name=fake.last_name(),
+                password=fake.password(),
+                is_active=True,
+                is_staff=False,
+                is_superuser=False,
+            )
+            user.save()
+
+        for space in spaces:
+            if random.random() < 0.5:  # Adjust this probability to control how many spaces each user is assigned
+                space.members.add(user)
+                space.moderators.add(user)
+                space.save()
+
+    # Create posts with your specified data
+    posts_data = [
```

Comment:
After this line the code is not indented as previously. Could you add on more tab to the code below please?